### ISSUE TITLE
Add support for MultiSubnetFailover

### ISSUE THREAD

**Issue by andrewl913** [prisma/prisma#9187]
## Problem
When using the new SQL server DB connector.  We would like to connect to the always-on availability group.

## Suggested solution

Add support as a connection string property for the SQL server to include`multiSubnetFailover=true`

## Additional context
https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql/sqlclient-support-for-high-availability-disaster-recovery

Thanks for all your hard work and support!

**Comment by kihonq** [comment]
Would love to see this option available too ! Thanks !

**Comment by salam-seek** [comment]
Facing the same issue as well, is there any workaround anyone can suggest ?

**Comment by finn-orsini** [comment]
👋 Hoping to get some eyes on this, the lack of this config is impacting our apps ability to connect reliably to our DB. 

Is it as easy as updating [the options here](https://github.com/prisma/prisma/blob/main/packages/client/src/utils/setupMSSQL.ts#L21-L24) to accept this value from the connection string?

**Comment by dariusj18** [comment]
Bump, seems a pretty important enterprise feature to be lacking.

**Comment by pip8786** [comment]
@andrewl913 or others on this thread, what do you see when you connect prisma to a server with multi subnet failover turned on? We're getting timeouts getting a connection seemingly randomly since we switched to a server with this on and are trying to determine if this is the issue.

**Comment by dariusj18** [comment]
@pip8786 yes, timeouts are the issue.  without multisubnetfailover the driver (given multiple results in a DNS query) will attempt to connect to the first one identified.  This will timeout if the first server it is trying is not accessible.  The multisubnetfailover flag tells the driver to attempt to connect to each server in parallel and go with whichever it succeeds connecting to first.

**Comment by pip8786** [comment]
@dariusj18 Thanks for clarifying. I've added the issue to the driver repo in case that gains more traction. Looks like you saw it too. Hopefully they can fix it so we don't have to move off of prisma.

**Comment by hirdstclendenin** [comment]
Now that Prisma has a client adapter for mssql using tedious, this should be as easy as allowing the parameter MultiSubnetFailover through from the adapter connection string. (I think this is working with the config passed as options)

Feature supported via connection options 

https://github.com/tediousjs/tedious/blob/ebb023ed90969a7ec0e4b036533ad52739d921f7/src/connection.ts#L756

Any idea of when the driver adapters will be working with the Prisma migrations? I don't thin this will really be resolved until the Prisma config file can be used to set an adapter for MSSQL that supports `SqlMigrationAwareDriverAdapterFactory`
